pkg/KubeArmorOperator/cmd/operator/root.go

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+// Package cmd is the collection of all the subcommands available in kArmor while providing relevant options for the same
+package operator
+
+import (
+	"errors"
+	"path/filepath"
+
+	opv1client "github.com/kubearmor/KubeArmor/pkg/KubeArmorOperator/client/clientset/versioned"
+	controllers "github.com/kubearmor/KubeArmor/pkg/KubeArmorOperator/internal/controller"
+	"github.com/kubearmor/KubeArmor/pkg/KubeArmorOperator/k8s"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/util/homedir"
+)
+
+var K8sClient *kubernetes.Clientset
+var Logger *zap.SugaredLogger
+var KubeConfig string
+var Context string
+var LsmOrder string
+var PathPrefix string
+var DeploymentName string
+var ExtClient *apiextensionsclientset.Clientset
+var Opv1Client *opv1client.Clientset
+
+// rootCmd represents the base command when called without any subcommands
+var Cmd = &cobra.Command{
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		log, _ := zap.NewProduction()
+		Logger = log.Sugar()
+		K8sClient = k8s.NewClient(*Logger, KubeConfig)
+		ExtClient = k8s.NewExtClient(*Logger, KubeConfig)
+		Opv1Client = k8s.NewOpv1Client(*Logger, KubeConfig)
+		//Initialise k8sClient for all child commands to inherit
+		if K8sClient == nil {
+			return errors.New("could'nt create k8s client")
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		nodeWatcher := controllers.NewClusterWatcher(K8sClient, Logger, ExtClient, Opv1Client, PathPrefix, DeploymentName)
+		go nodeWatcher.WatchConfigCrd()
+		nodeWatcher.WatchNodes()
+
+	},
+	Use:   "kubearmor-operator",
+	Short: "An operator to install kubearmor on k8s clusters",
+	Long: `An operator to install kubearmor on k8s clusters
+	
+KubeArmor is a container-aware runtime security enforcement system that
+restricts the behavior (such as process execution, file access, and networking
+operation) of containers at the system level.
+	`,
+	SilenceUsage:  true,
+	SilenceErrors: true,
+}
+
+func init() {
+	if home := homedir.HomeDir(); home != "" {
+		Cmd.PersistentFlags().StringVar(&KubeConfig, "kubeconfig", filepath.Join(home, ".kube", "config"), "Path to the kubeconfig file to use")
+	} else {
+		Cmd.PersistentFlags().StringVar(&KubeConfig, "kubeconfig", "", "Path to the kubeconfig file to use")
+	}
+	Cmd.PersistentFlags().StringVar(&LsmOrder, "lsmorder", "bpf,apparmor,selinux", "lsm preference order to use")
```

Comment:
```suggestion
	Cmd.PersistentFlags().StringVar(&LsmOrder, "lsm", "bpf,apparmor,selinux", "lsm preference order to use")
```
Let's keep consistency with Kubearmor's flags.